pkg/model/model.go

```diff
@@ -179,10 +184,10 @@ func (r TaggedResource) FilterThroughTags(filterTags []Tag) bool {
 	for _, resourceTag := range r.Tags {
 		for _, filterTag := range filterTags {
 			if resourceTag.Key == filterTag.Key {
-				r, _ := regexp.Compile(filterTag.Value)
-				if r.MatchString(resourceTag.Value) {
-					tagMatches++
+				if !filterTag.Value.MatchString(resourceTag.Value) {
+					return false
 				}
+				tagMatches++
```

Comment:
though subtle, can we add a note that this implcitly checking that all filtered key are present in the resource?

Reply:
This actually threw me off originally too and I almost did a bad refactor on it. A comment is absolutely  warranted

Reply:
Haha yhea, had to stare at the code for a bit saying wait, this aint necessary